LocalFeeder/server.py

```diff
@@ -49,7 +51,7 @@ def read_root():
     return JSONResponse(response, 200)
 
 
-@app.get("/sensor/")
+@app.get("/sensor")
 async def sensor():
     logging.info(os.getcwd())
     sensor_path = os.path.join(base_path, "sensors", "sensors.json")
```

Comment:
I'm unable to add a comment on the right section, but I noticed a couple problems caused by this function when trying to run with SMART-DS as well as deleting the existing sensors.json file
- We don't even use sensors.json
- This should almost certainly use asyncio.sleep(1) so that it doesn't block

Reply:
will have to resolve this issue once we have sensors published on the helics interface